### PR TITLE
(everything) new package

### PR DESCRIPTION
**Pull request by majkinetor**
Check it out (#443).

Notice the [Convert-ToEmbeded](https://github.com/chocolatey/chocolatey-coreteampackages/blob/9dc73656c3384c017e20957a3a3ced12dda8c71b/automatic/everything/update.ps1#L21-L48) function and how it is used. Perhaps somebody else will find it useful and if so, I may add it to AU.

### CODE REVIEW

File: automatic/everything/Everything.nuspec [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    <docsUrl>https://www.voidtools.com/faq/</docsUrl>
+    <mailingListUrl>https://www.voidtools.com/forum/</mailingListUrl>
+    <bugTrackerUrl>https://www.voidtools.com/forum/viewforum.php?f=6</bugTrackerUrl>
+    <packageSourceUrl>https://github.com/chocolatey/chocolatey-coreteampackages/tree/master/automatic/everything</packageSourceUrl>
+    <dependencies>
+      <dependency id="chocolatey-core.extension" version="1.0.2" />
+    </dependencies>
+
+  </metadata>
+</package>
```
Comment (AdmiringWorm) [inline automatic/everything/Everything.nuspec]: looks like you're missing the files element

**Comment by AdmiringWorm** [comment]
btw, I found IMO a simpler way to download the files that could perhaps be used.
I tested out in my own repository reusing chocolatey helpers (namely `Get-WebFileName` and `Get-WebFile`), and found this to be quite simple.

I'm currently using that method for the InnoSetup package as can be seen here:
https://github.com/AdmiringWorm/chocolatey-packages/blob/master/automatic/InnoSetup/update.ps1

File: automatic/everything/Everything.nuspec (hunk 1 above)
Comment (majkinetor) [inline automatic/everything/Everything.nuspec]: looks like it isn't needed ...

Comment (AdmiringWorm) [inline automatic/everything/Everything.nuspec]: if you don't have the files element, it will also pack the `update.ps1` file

**Pull request merged**